### PR TITLE
RCHAIN-3582 change pyrchain protocol

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
change pyrchain protocol

### JIRA ticket:
change pyrchain protocol according RCHAIN-3582

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by zsluedem** [comment]
bors try

**Review by woky — APPROVED** [review]
LGTM

**Review by KentShikama — APPROVED** [review]

File: integration-tests/test/node_client.py [hunk 1]
```diff
@@ -168,12 +174,15 @@ def _start_transport_server(self) -> grpc.Server:
...
     def block_request(self, block_hash: str, rnode: RNode) -> BlockMessage:
-        block_request = BlockRequest(base16Hash=block_hash, hash=base64.b16decode(block_hash, True))
+        block_request = BlockRequest(hash=bytes.fromhex(block_hash))
         request_msg_packet = Packet(typeId="BlockRequest", content=block_request.SerializeToString())
         protocol = Protocol(header=self.header_pb, packet=request_msg_packet)
         request = TLRequest(protocol=protocol)
         self.send_request(request, rnode)
-        return self.return_queue.get(timeout=20)
+        try:
+            return self.return_queue.get(timeout=20)
```
Comment (adaszko) [inline integration-tests/test/node_client.py]: 20 is a magic value here.  It's prolly better to use for that.

Comment (zsluedem) [reply integration-tests/test/node_client.py]: Do you think it is more appropriate to use the timeout in the context?

Comment (adaszko) [reply integration-tests/test/node_client.py]: Sorry, I ate a word.  I meant `receive_timeout` in place of 20.

**Pull request merged**